source/documentation/deploying_services/route_services.md

```diff
@@ -47,14 +49,14 @@ This is how you define an user-provided route service instance and map it to the
     ```
 
     where:
-    - `DOMAIN` is the domain used by your application, likely the default one `cloudapps.digital`
+    - `DOMAIN` is your app domain, and varies depending on which [region](/orgs_spaces_users.html#regions) your app is hosted in
     - `SERVICE_INSTANCE` the name of the service that you have just created
     - `HOSTNAME` the host or app name assigned to the app
 
-    For example:
+    For example, if your app is named `myapp` and is hosted in the London region:
 
     ```
-    cf bind-route-service cloudapps.digital my-route-service --hostname myapp
+    cf bind-route-service london.cloudapps.digital my-route-service --hostname myapp
```

Comment:
These `london` bits seem unrelated to the internal apps PR - are they deliberate?